src/ripple/consensus/Consensus.cpp

```diff
@@ -49,21 +64,24 @@ shouldCloseLedger(
         return true;
     }
 
-    if ((proposersClosed + proposersValidated) > (prevProposers / 2))
+    if (prevProposers >= 1)
     {
-        // If more than half of the network has closed, we close
-        JLOG(j.trace()) << "Others have closed";
-        return true;
+        if ((proposersClosed + proposersValidated) > (prevProposers / 2))
```

Comment:
I agree that the comment does not match the check. I've avoided changing it for a while, but I think your suggestion is reasonable. However, it does require reworking the unit tests a bit since it changes some of the close time dynamics there. I'd rather have it as a separate changeset.

@JoelKatz, do you have a preference?

Reply:
Rather than outlining all scenarios, I think the basic principle is simply that if you haven't been able to advance the fully validated ledger, be willing to wait a bit longer before the next deliberation phase. This provides more time to hear enough validations to increase be sure you are working on the proper prior ledger. It also should reduce the load on the network during this period when things are already degraded.

However, your other points about how to treat  this clause have led me back to the drawing board. I  think the proper approach is to base the gap on the preferred ledger sequence number, rather than the last fully validated one.